src/models/mcs_def.jl

```diff
@@ -2,161 +2,135 @@ function getsim()
+        SeaLevelRise.sltau_SLresponsetime = Gamma(16.0833333333333333, 24.)                      # fat-tailed distribution of time constant T_sl, sea level response time, from mode=362, mean = 386
 
         # GDP
-        isat0_initialimpactfxnsaturation = TriangularDist(15, 25, 20)
+        GDP.save_savingsrate = RV_save_savingsrate
+        GDP.isat0_initialimpactfxnsaturation = TriangularDist(15, 25, 20)
 
         # MarketDamages
-        iben_MarketInitialBenefit = TriangularDist(0, .3, .1)
-        W_MarketImpactsatCalibrationTemp = TriangularDist(.2, .8, .5)
-        pow_MarketImpactExponent = TriangularDist(1.5, 3, 2)
-        ipow_MarketIncomeFxnExponent = TriangularDist(-.3, 0, -.1)
+        MarketDamages.tcal_CalibrationTemp = RV_tcal_CalibrationTemp
+        MarketDamages.save_savingsrate = RV_save_savingsrate
+        MarketDamages.iben_MarketInitialBenefit = TriangularDist(0, .3, .1)
+        MarketDamages.W_MarketImpactsatCalibrationTemp = TriangularDist(.2, .8, .5)
+        MarketDamages.pow_MarketImpactExponent = TriangularDist(1.5, 3, 2)
+        MarketDamages.ipow_MarketIncomeFxnExponent = TriangularDist(-.3, 0, -.1)
 
         # MarketDamagesBurke
-        impf_coeff_lin = TriangularDist(-0.0139791885347898, -0.0026206307945989, -0.00829990966469437)
-        impf_coeff_quadr = TriangularDist(-0.000599999506482576, -0.000400007300924579, -0.000500003403703578)
-
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["EU"] = TriangularDist(6.76231496767033, 13.482086163781, 10.1222005657257)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["USA"] = TriangularDist(9.54210085883826, 17.3151395362191, 13.4286201975287)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["OECD"] = TriangularDist(9.07596053028087, 15.0507477943984, 12.0633541623396)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["USSR"] = TriangularDist(3.01320548016903, 11.2132204366259, 7.11321295839747)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["China"] = TriangularDist(12.2330402806912, 17.7928749427573, 15.0129576117242)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["SEAsia"] = TriangularDist(23.3863348263352, 26.5136231383473, 24.9499789823412)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["Africa"] = TriangularDist(20.1866940491107, 23.5978086497453, 21.892251349428)
-        MarketDamagesBurke_rtl_abs_0_realizedabstemperature["LatAmerica"] = TriangularDist(19.4846849750102, 22.7561130637973, 21.1203990194037)
+        MarketDamagesBurke.save_savingsrate = RV_save_savingsrate
+        MarketDamagesBurke.impf_coeff_lin = TriangularDist(-0.0139791885347898, -0.0026206307945989, -0.00829990966469437)
+        MarketDamagesBurke.impf_coeff_quadr = TriangularDist(-0.000599999506482576, -0.000400007300924579, -0.000500003403703578)
 
         # NonMarketDamages
-        tcal_CalibrationTemp = TriangularDist(2.5, 3.5, 3.)
```

Comment:
this was previously getting assigned twice

Reply:
Thanks.